### ISSUE TITLE
Allow manual addition and deletion of event and property definitions

### ISSUE THREAD

**Issue by Twixes** [PostHog/posthog#6099]
## Is your feature request related to a problem?

We currently show event and property definitions on the Events page (tabs Events Stats and Properties Stats), which are then used in event/property selection dropdowns. However, these are only based on incoming data, and sometimes I'd like to be build appropriate queries _before_ PostHog sees that event/property.

## Describe the solution you'd like

It should simply be possible to create and delete definitions manually on the Events page.

## Additional context

This would solve a problem reported by a user: https://app.papercups.io/conversations/all?cid=2bfed59e-c816-4a52-b8c6-68fa99b7d700

**Comment by weyert** [comment]
Yes, the use case for me is that I want to define feature flags filters against user properties while I don't have access to the targeted site due it being a production environment. 

During the chat session it was suggested I could identify the user with the needed user property via the API but then I would need to send a test event. Only this test event can't be deleted afterwards. 

I will try if I can define such feature flag (or update its filters) via the API; not sure if that would cause any problems in the backend @Twixes

**Comment by darkopia** [comment]
+1 https://posthoghelp.zendesk.com/agent/tickets/24285 (moved to PostHog: https://us.posthog.com/project/2/support/tickets/26499)

**Comment by luke-belton** [comment]
+1 [#33180](https://posthoghelp.zendesk.com/agent/tickets/33180) (moved to PostHog: https://us.posthog.com/project/2/support/tickets/35703)
